system/View/Filters.php

```diff
@@ -208,7 +208,7 @@ public static function prose(string $value): string
      *  - ceil      always rounds up
      *  - floor     always rounds down
      *
-     * @param mixed $precision
+     * @param float|int|string $precision
```

Comment:
Precision should never be a float.
```suggestion
     * @param int|string $precision
```

Reply:
Done